src/kaleidoscope/key_defs_keymaps.h

```diff
@@ -21,6 +21,7 @@
 #include "kaleidoscope_internal/deprecations.h"
 
 static const uint8_t LAYER_SHIFT_OFFSET = 42;
+static const uint8_t LAYER_MOVE_OFFSET = LAYER_SHIFT_OFFSET + 33;
```

Comment:
What's with the 33?

Reply:
(I presume "we are assuming no more than 32 layers")
But in that case, shift and move offsets should be the same number. and the 42 should probably get lifted to a named constant taht explains itself.

Reply:
Yes, it's no more than 32 layers. I'll use 42 as the offset for move as well then. (If I could go back in time, I'd use 33, but I can only warp through the screen, not time :/)

As for the name, how about `LAYER_OP_OFFSET`? Or just `LAYER_OFFSET`...

Reply:
LAYER_OP_OFFSET feels less likely to step on something else outside of the scope of this file